functions.php

```diff
@@ -218,37 +218,37 @@ function odm_jeo_scripts()
   endif;
 
   if (is_page( array( 'map-explorer', 'embed' )) || is_singular('map') || is_singular('map-layer') || is_singular('profiles') || is_home()):
-    
+
     if ( file_exists( STYLESHEETPATH . '/inc/jeo-scripts/jeo.js')):
        wp_deregister_script('jeo');
        wp_enqueue_script('jeo', get_stylesheet_directory_uri() . '/inc/jeo-scripts/jeo.js', array('mapbox-js', 'underscore', 'jquery'), '0.5.0');
     endif;
 
-    if ( file_exists( STYLESHEETPATH . '/inc/jeo-scripts/fullscreen.js')):
-       wp_deregister_script('jeo.fullscreen');       
+    if ( file_exists( STYLESHEETPATH . '/inc/jeo-scripts/control_fullscreen.js')):
```

Comment:
Most of jeo functions has been modified. the parent's script should deregister, otherwise it will affect to the presentation of the map layers.